src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -84,6 +85,10 @@ public class ExtensionsRunner {
     private final TransportInterceptor NOOP_TRANSPORT_INTERCEPTOR = new TransportInterceptor() {
     };
     private NamedWriteableRegistryAPI namedWriteableRegistryApi = new NamedWriteableRegistryAPI();
+    /*
+     * TODO: expose an interface for extension to register actions
+     */
+    private TransportActionsAPI transportActionsAPI = new TransportActionsAPI(new HashMap<>());
```

Comment:
Are we following the naming convention ending with `API`?

Reply:
I think we decided to try to avoid using API/Api altogether.

Reply:
My thoughts, anything SDK exposes to the Extension is an API.
If you think that's still confusing, what are your suggestions ? @owaiskazi19 @dbwiddis 
May be just `TransportActions` ?

Reply:
I like TransportActions. 

also SDK == API and everything in the package ending in SDK should be automatically thought of as API. (Runner and interfaces at present.)

Reply:
Changed it to Transport Actions.